blog/steering/2020-11-ksc-elections.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Democracy is a beautiful thing"
+linkTitle: "Democracy is a beautiful thing"
+Author: [Carlos Santana](https://twitter.com/csantanapr)
+Author handle: https://github.com/csantanapr
+date: 2020-11-06
+description: "Knative Steering Committee Elections"
+type: "blog"
+---
+
+Democracy is a beautiful thing, those that want to dedicate their own time to improve a community can step up and those that have the right to vote can freely speak their mind to elect the candidate they think will be a net positive to the community.
+
+We are running an election to fill out the two (2) seats due for election
+this year on the [Knative Steering Committee](https://knative.dev/community/contributing/steering-committee/). Each elected member will serve
+a two (2) year term.
+
+This election will shape the future of Knative as a community and project.
+The Knative Steering Committee (KSC) is
+responsible for the general health of the Knative community, and this is the first general election for that body.
+
+The process already started on October 27th with the announcement of the elections to the mailing list. The next milestone is to have a complete list of candidates, and nominations close on November 9th at 0000 UTC (5pm Pacific).
+
+If you want to stand for election, open a PR against the
+[knative/community repository](https://github.com/knative/community) to include
+your candidate profile in the `/elections/2020/SC` folder, with your name as the
+filename.  This profile should include:
+
+* Your name
+* Your company affiliation (employer or otherwise)
+* Your contributions to Knative
+* Why you are running
+
+You can find [a sample template in the folder](https://github.com/knative/community/blob/master/elections/2020/SC/candidate-template.md).
+
+
+This is the schedule for the elections if you want to follow along.
+
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| October 27   | Announcement of Election, call for nominations |
+| November 9   | All candidate nominations due by 0000 UTC (5pm Pacific) |
+| November 12  | Election Begins via email ballots |
+| November 22  | Last call for exception & replacement ballots |
+| November 29  | Election Closes by 0000 UTC (5pm Pacific) |
+| December 3   | Announcement of Results at Knative Steering Meeting |
+
+
+You can read the current [Knative Steering Committee Election - 2020 Voters Guide](https://github.com/knative/community/blob/master/elections/2020/SC/README.md) for all the details.
+
+And remember if you have the right to vote, speak your mind and vote because every vote counts!
```

Comment:
Sorry to comment on the linting check below, but I really wonder who was ever hurt by a trailing newline of a document so much that it justifies an extra turn around ? It's not nice, sure, but was is the reasoning that a missing newline at the end of a document needs to break the build ? Which modern editor has an issue with those or even expose this as an error ? Really curious, as I have seen that now several times and IMO it's just a waste to time to fix this (or write these kind of comments). tbh, most of the "markdown checks" are much more harmful than helpful. It's prose.

Reply:
I'd put a lot of lint checks in that boat -- particularly ones that could either be automatic fixes or just ignored.